bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -205,32 +204,7 @@ protected ToolbarSlideout createSlideout(final ToolBar toolbar) {
 
    private void addPartListener() {
 
-      _partListener = new IPartListener2() {
-
-         @Override
-         public void partActivated(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partBroughtToTop(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partClosed(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partDeactivated(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partHidden(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partInputChanged(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partOpened(final IWorkbenchPartReference partRef) {}
-
-         @Override
-         public void partVisible(final IWorkbenchPartReference partRef) {}
-      };
+      _partListener = new IPartListener2() {};
```

Comment:
When the part listener is empty, then there is no need to set a part listener at all -> remove `addPartListner`

Reply:
Ok, good to know. Any reason why it was here in the first place ? It's confusing to see something in the code if it's not used. 
Removed

Reply:
> Any reason why it was here in the first place ?

I propably copied an existing view and there was no need to use the part listener

Reply:
That would make sense. I removed all of them to remove confusion